lib/Word_Lib/Bitwise.thy

```diff
@@ -148,7 +148,7 @@ proof (cases "n < size w")
   case True
   then show ?thesis
     by (simp add: bl_sshiftr takefill_last_def word_size takefill_alt
-           rev_take last_rev drop_nonempty_def)    
+           rev_take last_rev drop_nonempty_def)
```

Comment:
yuck, yuck, yuck... is there any way you can force upstream to get their trailing whitespace sorted out? next we're going to see the return of tabs